libs/coin-evm/src/prepareTransaction.ts

```diff
@@ -127,7 +127,24 @@ export const prepareTransaction = async (
 ): Promise<EvmTransaction> => {
   const { currency } = account;
   // Get the current network status fees
-  const feeData = await getFeesEstimation(currency);
+  const feeData = await (async () => {
+    if (!transaction.feesStrategy) {
+      return await getFeesEstimation(currency);
+    }
+
+    if (transaction.feesStrategy === "custom") {
+      return {
+        gasPrice: transaction.gasPrice ?? null,
+        maxFeePerGas: transaction.maxFeePerGas ?? null,
+        maxPriorityFeePerGas: transaction.maxPriorityFeePerGas ?? null,
+      };
+    }
+
+    const gasOption = transaction.gasOptions?.[transaction.feesStrategy];
+
+    return gasOption ?? (await getFeesEstimation(currency));
```

Comment:
Same here, I think `return gasOption ?? getFeesEstimation(currency)` is equivalent ?